decidim-admin/app/models/decidim/admin/concerns/user_extends.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require "active_support/concern"
+
+module Decidim
+  module Admin
+    # A module that adds functionalities to the Decidim::User model class.
+    module UserExtends
+      extend ActiveSupport::Concern
+
+      included do
+        has_many :participatory_process_user_roles, foreign_key: "decidim_user_id", class_name: Decidim::ParticipatoryProcessUserRole
+        has_many :participatory_processes, through: :participatory_process_user_roles
```

Comment:
I would't do that, as:

1. This is glorified monkey-patching. We could be breaking features in core by introducing validation errors, conflicting names, etc. and we're clearly coupling the user to an ever-growing set of external needs. We have to find a different strategy: clearly, that's what a `Rectify::Query` is for.
2. `participatory_processes` isn't a good name, as they're actually *participatory processes that can be somehow managed by this user*. What if we want to have *participatory processes created by this user*? See 1)
3. There's no actual need to start the query from the user - we could find this by directly querying `Decidim::ParticipatoryProcessUserRoles` having a user instance. See 1).